src/content/en/2019/accessibility.md

```diff
@@ -245,40 +247,43 @@ The relative popularity of the [dialog role](https://developer.mozilla.org/en-US
+Unfortunately, 34.62% of sites have duplicate IDs, which means on many sites the ID specified by the user could refer to multiple different inputs. So when a user clicks on the label to select a field, they may end up [selecting something different](https://www.deque.com/blog/unique-id-attributes-matter/) than they intended. As you might imagine, this could have negative consequences in something like a shopping cart.
 
-This issue is even more pronounced for screen readers because their users may not be able to visually double check what is selected. Plus, many ARIA attributes, such as [`aria-describedby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute)  and [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute), work similarly to the label element detailed above. So, to make your site accessible, removing all duplicate IDs is a good first step.
+This issue is even more pronounced for screen readers because their users may not be able to visually double check what is selected. Plus, many ARIA attributes, such as [`aria-describedby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute)  and [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute), work similarly to the label element detailed above. So to make your site accessible, removing all duplicate IDs is a good first step.
 
 ## Conclusion
 
 People with disabilities are not the only ones with accessibility needs. For example, anyone who has suffered a temporary wrist injury has experienced the difficulty of tapping small tap targets. Eyesight often diminishes with age, making text written in small fonts challenging to read. Finger dexterity is not the same across age demographics, making tapping interactive controls or swiping through content on mobile websites more difficult for a sizable percentage of users.
 
-Similarly, assistive software is not only geared towards people with disabilities. For instance, drivers would benefit from a screen reading functionality that, while they keep their eyes on the road, reads long pieces of text like news stories aloud. The recent popularity of voice assistance, both on mobile devices and in the home, has demonstrated that controlling a computing device using voice commands is desirable, and at times essential, for a large number of users. Voice commands like these used to only be an accessibility feature but are now turning into a mainstream product. Captions are enjoyed not only by people who cannot hear a video but also by people who want to watch a video in a loud restaurant or in a library.
+Similarly, assistive software is not only geared towards people with disabilities:
```

Comment:
This feels like an incomplete sentence and it's unclear what the following list would contain. Consider rephrasing.

Reply:
Updated in the latest commit. How does it sound?

Reply:
SGTM!